### PR TITLE
Add mixins to css

### PR DESCRIPTION
**Pull request by kammon10**
#### What's this PR do?
Added mixins to css

#### Where should the reviewer start?
_mix-ins.scss and index.scss
#### How should this be manually tested?
na

#### Any background context you want to provide?
na

#### What are the relevant tickets?
na

#### Screenshots (if appropriate)
na

#### Questions:
na

### CODE REVIEW

**Review by aliroemhildt — APPROVED** [review]

File: src/scripts.js [hunk 1]
```diff
@@ -14,7 +14,8 @@ import {
   domUpdates,
   dropdownElement,
   quantityInput,
-  filterTags
+  filterTags,
+  displayIngredientsNeeded
 } from './domUpdates';
```
Comment (cagallo) [inline src/scripts.js:20]: Good catch, needed this imported!

File: src/styles/_mix-ins.scss [hunk 2]
```diff
@@ -3,3 +3,14 @@
...
+@mixin wrap-rows() {
+  flex-wrap: wrap;
+  justify-content: flex-start;
+  overflow: auto;
+}
+
+@mixin size($height, $width) {
+  height: $height;
+  width: $width;
+}
```
Comment (cagallo) [inline src/styles/_mix-ins.scss:16]: another mix-in! looks great!

**Review by cagallo — APPROVED** [review]

File: src/domUpdates.js [hunk 3]
```diff
@@ -161,7 +161,7 @@ function displayIngredientsNeeded(recipe) {
         <p>${matchedId.name}: ${ingredient.amount} ${ingWithUnits.quantity.unit}</p><br>
       `
       return acc;
-    }, '');
+    }, '<h4>missing ingredients</h4>');
```
Comment (anewb87) [inline src/domUpdates.js:164]: Our reset file messed this up a bit so we needed to add this and the 146 p tag margin agove.

File: src/domUpdates.js [hunk 4]
```diff
@@ -239,6 +239,16 @@ function createTable() {
       <td class="button-column hidden-row"></td>
     </tr>`
   }
+  const minusButtons = document.querySelectorAll('.minus-js');
```
Comment (anewb87) [inline src/domUpdates.js:242]: Just about to circle back to this. As is, it didn't work. This is our next issue to tackle.

File: src/domUpdates.js [hunk 5]
```diff
@@ -396,9 +406,9 @@ let domUpdates = {
     });
   },
 
-  displayMessageDropdown() {
+  displayMessageDropdown(message) {
```
Comment (anewb87) [inline src/domUpdates.js:409]: Making it dynamic so we could show some more user error handling.

File: src/scripts.js [hunk 6]
```diff
@@ -14,7 +14,8 @@ import {
   domUpdates,
   dropdownElement,
   quantityInput,
-  filterTags
+  filterTags,
+  displayIngredientsNeeded
```
Comment (anewb87) [inline src/scripts.js:18]: Forgot one, and then remembered it.

File: src/styles/_mix-ins.scss [hunk 7]
```diff
@@ -3,3 +3,14 @@
   opacity: 0;
   transition: visibility 0s 1s, opacity 1s linear;
 }
+
+@mixin wrap-rows() {
```
Comment (anewb87) [inline src/styles/_mix-ins.scss:7]: mixin it uuuuuup!

**Review by anewb87 — APPROVED** [review]
So close to done we can smell it. And it smells better than teen spirit.

**Pull request merged**